src/test/java/net/sf/jsqlparser/util/validation/validator/StatementValidatorTest.java

```diff
@@ -53,7 +61,8 @@ public void testValidateBlock() throws JSQLParserException {
     @Test
```

Comment:
Next time please use a Parametrised Test for this. Loops in Tests are an Anti-Pattern because you won't know which one failed. You can lookup the KeywordTests for getting a template.